content/awell-orchestration/api-reference/queries/get-data-point-definitions.mdx

```diff
@@ -31,31 +31,33 @@ query GetPathwayDataPointDefinitions(
     release_id: $release_id
   ) {
     data_point_definitions {
-      id
-      title
-      key
-      category
-      valueType
-      possibleValues {
-        value
-        label
+      id # unique identifier
+      title # string
+      key # string
+      category # CALCULATION, FORM, PATIENT_PROFILE, PATHWAY, TRACK, STEP
+      valueType # STRING, NUMBER, DATE, BOOLEAN, NUMBERS_ARRAY
+      possibleValues { # only for STRING value type
+        value # string
+        label # string
       }
       range {
-        min
-        max
+        min # number
+        max # number
       }
-      unit
-      optional
-      pii
-      metadata {
-        value
-        key
+      unit # string
+      optional # boolean
+      pii # boolean
+      data_point_metadata { # list of key-value objects
+        key # string
+        value # string
```

Comment:
@rahulkeerthi I am not too sure about all of the comments we place after the fields. It makes the query looks a bit messy and hard to read, and when I want to have a "clean" copy then I have to delete all the comments manually. 

Additionally, GraphQL is a typed query language so the added benefit of manually highlighting the types seems limited. They can always have a look at the schema to understand the types.

I do like the additional paragraph on data point metadata.